file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,592 @@
+        "! Semantic field type
+        "! $required
+        "! $maxLength 10
+        "! $values {@link zif_aff_cfdf_v1.data:gc_type}
+        semantic_field_type        TYPE ty_field_type,
+
+        "! <p class="shorttext">Length</p>
+        "! Length
+        "! $maximum 500
+        length                     TYPE i,
+
+        "! <p class="shorttext">Scale</p>
+        "! Scale
+        "! $maximum 14
+        scale                      TYPE i,
+
+        "! <p class="shorttext">Aggregation</p>
+        "! Aggregation
+        "! $maxLength 5
+        "! $values {@link zif_aff_cfdf_v1.data:gc_aggregation}
+        "! $default {@link zif_aff_cfdf_v1.data:gc_aggregation.none}
+        aggregation                TYPE ty_aggregation,
+
+        "! <p class="shorttext">Code List Uppercase Only</p>
+        "! Code List uppercase only
+        code_list_uppercase_only   TYPE abap_bool,
+
+        "! <p class="shorttext">Associated Business Object</p>
+        "! Associated business object
+        "! $maxLength 30
+        associated_business_object TYPE c LENGTH 30,
+
+        "! <p class="shorttext">Value Help View</p>
+        "! Value help view
+        "! $maxLength 30
+        value_help_view            TYPE ty_cds_view_name,
+
+        "! <p class="shorttext">Dimension View</p>
+        "! Dimension view
+        "! $maxLength 30
+        dimension_view             TYPE ty_cds_view_name,
+      END OF type,
+
+      "! <p class="shorttext">Data Protection and Privacy</p>
+      "! Data protection and privacy
+      data_protection_and_privacy TYPE ty_gs_dpp_properties,
+
+      "! <p class="shorttext">Reference to Characteristic</p>
+      "! Reference to characteristic
+      characteristic_reference    TYPE ty_gs_characteristic_reference,
+
+      "! <p class="shorttext">Label and Tooltip</p>
+      "! Label and tooltip
+      ui_texts                    TYPE ty_gt_ui_text,
+
+      "! <p class="shorttext">Business Contexts</p>
+      "! Business contexts
+      business_contexts           TYPE ty_gt_business_context,
+
+      "! <p class="shorttext">Code List Values</p>
+      "! Code list values
+      code_list_values            TYPE ty_gt_codelistvalue,
+
+      "! <p class="shorttext">Code List Descriptions</p>
+      "! Code list descriptions
+      code_list_descriptions      TYPE ty_gt_codelistvalue_descr,
+
+      "! <p class="shorttext">Value Help Key Mappings</p>
+      "! Value help key mappings
+      value_help_key_mappings     TYPE ty_gt_value_help_key_mapping,
+
+      "! <p class="shorttext">Value Help Bindings</p>
+      "! Value help bindings
+      value_help_bindings         TYPE ty_gt_value_help_binding,
+
+      "! <p class="shorttext">Extended CDS Views</p>
+      "! Extended CDS views
+      cds_usages                  TYPE ty_gt_cds_usage,
+
+      "! <p class="shorttext">Extended OData Services</p>
+      "! Extended OData services
+      odata_usages                TYPE ty_gt_odata_usage,
```

Comment:
If the title does not fit to the field names it might be an indicator to rename one of them ;)

Same for dynpro and soap usages...

Reply:
Done